vioscsi/helper.c

```diff
@@ -82,53 +86,55 @@ ENTER_FN_SRB();
             }
         } else {
             RhelDbgPrint(TRACE_LEVEL_ERROR, " StorPortGetStartIoPerfParams failed srb 0x%p status 0x%x MessageNumber %d.\n", Srb, status, param.MessageNumber);
+            // FIXME
+            // Should we return on this error..?
+            // Should it be TRACE_LEVEL_FATAL..?
+            //is_done = TRUE;
         }
     }
-
+    
     srbExt = SRB_EXTENSION(Srb);
-
+    
     if (!srbExt) {
         RhelDbgPrint(TRACE_LEVEL_INFORMATION, " No SRB Extenstion for SRB 0x%p \n", Srb);
+EXIT_FN_SRB();
         return;
     }
-
+    
     MessageID = QUEUE_TO_MESSAGE(QueueNumber);
     index = QueueNumber - VIRTIO_SCSI_REQUEST_QUEUE_0;
-
+    
     if (adaptExt->reset_in_progress) {
-        RhelDbgPrint(TRACE_LEVEL_FATAL, " Reset is in progress, completing SRB 0x%p with SRB_STATUS_BUS_RESET.\n", Srb);
+        RhelDbgPrint(TRACE_LEVEL_WARNING, " Reset is in progress, completing SRB 0x%p with SRB_STATUS_BUS_RESET.\n", Srb);
```

Comment:
I'm presuming that we have historically always wanted to see this. In WPP, I think this would be TRACE_LEVEL_NONE, but in Dbg this probably wouldn't work. However, it is actually just a warning, and should succeed following the reset. If not, this should generate an TRACE_LEVEL_ERROR or TRACE_LEVEL_FATAL event.